Arm/rover_arm_system.hpp

```diff
@@ -3,58 +3,199 @@
 #include "joint.hpp"
 #include "Hand/wrist_joint.hpp"
 #include "../Common/rover_system.hpp"
+#include <cmath>
 
 namespace sjsu::arm
 {
 class RoverArmSystem : public sjsu::common::RoverSystem
 {
- private:
-  // isOperational defines if the arm should be allowed to move or if it should
-  // be in its resting 'off' position.
-  bool isOperational;
-
-  sjsu::arm::Joint & Rotunda;
-  sjsu::arm::Joint & Shoulder;
-  sjsu::arm::Joint & Elbow;
-  sjsu::arm::WristJoint & Wrist;
-
-  // The target angle for each of the joints
-  units::angle::degree_t shoulder_pos;
-  units::angle::degree_t elbow_pos;
-  units::angle::degree_t rotunda_pos;
-  units::angle::degree_t wrist_roll_pos;
-  units::angle::degree_t wrist_pitch_pos;
-
  public:
+  struct MissionControlData
+  {
+    enum class Modes : char
+    {
+      kDefault = 'D',
+    };
+    int is_operational;
+    int heartbeat_count = 0;
+    Modes modes         = Modes::kDefault;
+    int rotunda_speed;
+    int rotunda_angle;
+    int shoulder_speed;
+    int shoulder_angle;
+    int elbow_speed;
+    int elbow_angle;
+    int wrist_speed;
+    int wrist_roll;
+    int wrist_pitch;
+    struct Fingers
+    {
+      int pinky;
+      int ring;
+      int middle;
+      int pointer;
+      int thumb;
+    };
+    Fingers fingers;
+  };
+
```

Comment:
Do we need speed variables? Is it important that arm moves at a certain speed? Or can this just be fixed to say 20rpm?